app/views/admin_general/_to_do_list.html.erb

```diff
@@ -38,7 +38,8 @@
                           <% end %>
                         </td>
                       </tr>
-                      <% if item.described_state == 'attention_requested' %>
+                      <% if item.described_state == 'attention_requested' ||
+                            item.last_event.event_type == 'send_error' %>
```

Comment:
We think it would reduce complexity if we made this more like the `params[:message]` conditional:

```erb
<% if item.last_event.params[:reason] %>
  <tr>
    <td><b>Reason</b></td>
    <td>
      <%= item.last_event.params[:reason] %>
    </td>
  </tr>
<% end %>
```